aea/protocols/dialogue/base.py

```diff
@@ -859,7 +868,7 @@ def _validate_message_target(self, message: Message) -> Optional[str]:
 
         if abs(target) > max(latest_ids):
             return "Invalid target. Expected a value less than or equal to abs({}). Found abs({}).".format(
-                self._last_message_id, target
+                max(latest_ids), abs(target)
```

Comment:
This seemed to have been a bug

Reply:
I think it should be `target`. The text reads should be less than `abs(max(latest_id))`. Found `abs(X)`. Here X must be `target` and not `abs(target)`, no?

Reply:
it's general check, so if  messages in dialoge is like [1, -1] and target is 2 or -2 it's bad
but if messages are [1,-1, 2] and target is -2 this check will pass, but on get_message_by_id it will fail